### ISSUE TITLE
Move Airflow v2 into production

### ISSUE THREAD

**Issue by mddilley** [cityofaustin/atd-data-tech#12453]
## Done

- [x] Readme https://github.com/cityofaustin/atd-airflow/pull/109
- [x] Slack notifier https://github.com/cityofaustin/atd-airflow/pull/102
- [x] 1Password util https://github.com/cityofaustin/atd-airflow/pull/107
- [x] Certbot renew script (current SSL cert expires on 8/15/2023) https://github.com/cityofaustin/atd-airflow/pull/106
- [x] CRIS import https://github.com/cityofaustin/atd-airflow/pull/108
- [x] Trail counters data publishing https://github.com/cityofaustin/atd-airflow/pull/103
- [x] Service bot https://github.com/cityofaustin/atd-airflow/pull/100
- [x] Collection branch for changes that require stack rebuild https://github.com/cityofaustin/atd-airflow/pull/105
- [x] https://github.com/cityofaustin/atd-data-tech/issues/12552
- [x] https://github.com/cityofaustin/atd-airflow/pull/113
- [x] https://github.com/cityofaustin/atd-data-tech/issues/12557
- [x] Setup Github CI for docker? https://github.com/cityofaustin/atd-airflow/pull/113
- [x] review webhook code
- [x] make sure the team understands how this works and that it's documented? (added this to https://github.com/cityofaustin/atd-data-tech/issues/12454)
- [x] make sure the team understand why we need this and how it works (added this to https://github.com/cityofaustin/atd-data-tech/issues/12454)
- [x] Get Slack notifier merged
  - this blocks all three DAGs from merging since we need failure notifications
- [x] Get 1Password util merged
  - ~this is blocked by resolving the 1PW vault organization discussion~ from backlog refinement on 6/7/2023: going forward with where we are at
  - this could block all three DAGs from merging (we could refactor later or just use 1PW util going forward)
- [x] Get Cerbot renew script merged
  - this is lower priority since we have the ability to run without SSL but should be done before current cert expires
- [x] Get readme and PR template PR merged 
- [x] Clean up `/usr/airflow/` on atd-data03 (we still have some directories besides `atd-airflow` from the process of standing this up)
- [x] configure `atd-airflow` docker CI action
- [x] Add note in readme or Haproxy config that explains that Flower auth is only required in prod
- [x] https://github.com/cityofaustin/atd-data-tech/issues/12654 @frankhereford 
- [x] Get stack update collection branch merged into `production` and "rebuild" and deploy on atd-data03
- [x] test production haproxy config on atddata03
- [x] Merge DAG PRs listed above after they have at least the Slack notifier integrated
- [x] Create fernet and add key to production env and 1password @frankhereford 
- [x] Fully terminate and start production docker stack to install fernet key and mount postgres volume
- [x] install certbot `cron` job on atd-data03 (Frank proposed sym-linking a cron.d file into system cron directory) @mddilley 
- [x] Change repo default branch to `production` and add branch protection.
- [x] https://github.com/cityofaustin/atd-data-tech/issues/12723
- [x] remove/revoke the airflow deploy key that was created for the webhook

## Todo
- [x] what flows need to migrate?
- [x]  there's an SMTP server for airflow v1 in aws. see "Airflow SMPT" entry in 1pass. config will be removed when we shut 
- [x] Create read-only account for service desk/non dev users @johnclary 
- [x] Merge / close the Airflow v2 branch. Charlie's trail counter's DAG is in there! @Charlie-Henry
- [x] #12853

down airflow v1
- https://github.com/cityofaustin/atd-data-tech/issues/12804
- https://github.com/cityofaustin/atd-data-tech/issues/12736
- https://github.com/cityofaustin/atd-data-tech/issues/12347
- https://github.com/cityofaustin/atd-data-tech/issues/12780
- https://github.com/cityofaustin/atd-data-tech/issues/12733 [icebox'd]

**Comment by frankhereford** [comment]
During our recent airflow meeting, we had a discussion about installing a script in the `atd-data03` cron system on behalf of the airflow system. We discussed this approach which intends to not only document the cron entry, but also to store the cron entry itself in a file tracked by git.

We might:
* Create the file inside the airflow repo, perhaps as `toolbox/certbot/cron_entry`
* That file might contain something akin to:

```
0 */12 * * * root /usr/airflow/atd-airflow/toolbox/certbot/renew.sh
```
* That file needs to be represented in `/etc/cron.d/`, so we could `sudo ln -s /usr/airflow/atd-airflow/toolbox/certbot/cron_entry /etc/cron.d/renew_airflow_ssl_certificates` such that the entry is picked up by the cron system which looks for arbitrary cron entries in configuration files in that folder.
